recipes/xo/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/scopatz/xo.git
+  git_tag: 8f486a2709d376d00f98c09a86179bce2a99099a
```

Comment:
😞  - are you likely to tag ```v0.1.11```? If so, I'd encourage you to download the prepared ```.tar.gz``` linked from there. If not, maybe you could use https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz.

That said, given the size of the repo, it isn't a big deal either way.